src/routes/userRoutes.ts

```diff
@@ -0,0 +1,13 @@
+import { Hono } from 'hono';
+import { userController } from '@/controllers/userController';
+import { UserBodySchema } from '@/validation/body-check';
+import { zValidator } from '@hono/zod-validator';
+import { validateUserId } from '@/middlewares/userValidation';
+
+export const userRoutes = new Hono().basePath('/users');
+
+userRoutes.get('/:id', validateUserId, userController.getById);
+userRoutes.get('/', userController.getAll);
+userRoutes.post('/', zValidator('json', UserBodySchema), userController.create);
```

Comment:
1. I can create a user with a space instead of a name. Please handle this case
2. Let’s force a user to create a harder password, it should include a letter in lower and upper cases, number and 3 special symbols. Currently I can create a user with ‘aaaaaaa’ password
3. Remove id from validation, it should be autogenerated. Because how we should know what an id is free on a front-end?